purchases_ui_flutter/lib/purchases_ui_flutter.dart

```diff
@@ -50,6 +49,13 @@ class RevenueCatUI {
     return _parseStringToResult(result);
   }
 
+  static presentCustomerCenter() async {
```

Comment:
I'm not totally sure if it would pick the correct type automatically, but should this be
```suggestion
  static Future<void> presentCustomerCenter() async {
```

Reply:
Surprised it worked without it 🤔